packages/xchain-cosmos/src/client.ts

```diff
@@ -322,39 +243,38 @@ class Client implements CosmosClient, XChainClient {
       memo,
     })
 
-    return transferResult?.txhash || ''
+    return transferResult || ''
   }
 
-  /**
-   * Transfer offline balances.
-   *
-   * @param {TxOfflineParams} params The transfer offline options.
-   * @returns {StdTx} The signed transaction.
-   */
-  async transferOffline({
-    walletIndex,
-    asset,
-    amount,
-    recipient,
-    memo,
-    from_account_number,
-    from_sequence,
-  }: TxOfflineParams): Promise<StdTx> {
-    const fromAddressIndex = walletIndex || 0
-    registerCodecs()
-
-    const mainAsset = this.getMainAsset()
-    return await this.getSDKClient().transferSignedOffline({
-      privkey: this.getPrivateKey(fromAddressIndex),
-      from: this.getAddress(fromAddressIndex),
-      from_account_number,
-      from_sequence,
-      to: recipient,
-      amount: amount.amount().toString(),
-      asset: getDenom(asset || mainAsset),
-      memo,
-    })
-  }
+  // /**
+  //  * Transfer offline balances.
+  //  *
+  //  * @param {TxOfflineParams} params The transfer offline options.
+  //  * @returns {StdTx} The signed transaction.
+  //  */
+  // async transferOffline({
+  //   walletIndex,
+  //   asset,
+  //   amount,
+  //   recipient,
+  //   memo,
+  //   from_account_number,
+  //   from_sequence,
+  // }: TxOfflineParams): Promise<StdTx> {
+  //   const fromAddressIndex = walletIndex || 0
+
+  //   const mainAsset = this.getMainAsset()
+  //   return await this.getSDKClient().transferSignedOffline({
+  //     privkey: this.getPrivateKey(fromAddressIndex),
+  //     from: this.getAddress(fromAddressIndex),
+  //     from_account_number,
+  //     from_sequence,
+  //     to: recipient,
+  //     amount: amount.amount().toString(),
+  //     asset: getDenom(asset || mainAsset),
+  //     memo,
+  //   })
+  // }
```

Comment:
`transferOffline` was added by the community - I think we need to keep it accessible, but not comment it.